openml/config.py

```diff
@@ -7,47 +7,94 @@
 import logging
 import logging.handlers
 import os
-from typing import cast
+from typing import cast, Optional, Tuple, Any
 
 from io import StringIO
 import configparser
 from urllib.parse import urlparse
 
 logger = logging.getLogger(__name__)
+openml_logger = logging.getLogger('openml')
+console_handler = None
+file_handler = None
 
 
-def configure_logging(console_output_level: int, file_output_level: int):
-    """ Sets the OpenML logger to DEBUG, with attached Stream- and FileHandler. """
-    # Verbosity levels as defined (https://github.com/openml/OpenML/wiki/Client-API-Standards)
-    # don't match Python values directly:
-    verbosity_map = {0: logging.WARNING, 1: logging.INFO, 2: logging.DEBUG}
-
-    openml_logger = logging.getLogger('openml')
-    openml_logger.setLevel(logging.DEBUG)
+def _create_log_handlers():
+    """ Creates but does not attach the log handlers. """
+    global console_handler, file_handler
```

Comment:
I suggest checking first if they are still `None`, and return early if they are not. So that calling this multiple times does nothing. Otherwise you might override some later changes done by people to original handlers.

Reply:
Good catch